webpack.config.js

```diff
@@ -0,0 +1,205 @@
+'use strict';
+
+// Modules
+var webpack = require('webpack');
+var autoprefixer = require('autoprefixer');
+var HtmlWebpackPlugin = require('html-webpack-plugin');
+var ExtractTextPlugin = require('extract-text-webpack-plugin');
+var CopyWebpackPlugin = require('copy-webpack-plugin');
+/**
+ * Env
+ * Get npm lifecycle event to identify the environment
+ */
+var ENV = process.env.npm_lifecycle_event;
+var isTest = ENV === 'test' || ENV === 'test-watch';
+var isProd = false;
+
+module.exports = function makeWebpackConfig () {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Wrap an immediate function invocation in parentheses.](https://www.codacy.com/app/pascal_2/openmrs-owa-conceptdictionary/file/2307931000/issues/source?bid=3196969&fileBranchId=3196971#l17)